contributors/devel/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
Should this file perhaps be at the `contributors/` level, and all the OWNER files in subfolders (like the ones in `contributors/design-proposals/`) be removed?